jenkins/jobs/ironic_image_build_test.pipeline

```diff
@@ -19,10 +19,10 @@ pipeline {
   agent { label 'metal3-static-workers' }
   environment {
     // PR, source and destination branch information
-    DESTINATION_REPO = "${PROJECT_REPO_ORG}/${PROJECT_REPO_NAME}"
-    DESTINATION_BRANCH = "${ghprbTargetBranch ?: main}"
-    SOURCE_REPO = "${ghprbAuthorRepoGitUrl ?: env.DESTINATION_REPO}"
-    SOURCE_BRANCH = "${ghprbActualCommit ?: env.DESTINATION_BRANCH}"
+    DESTINATION_REPO = "${env.REPO_OWNER}/${env.REPO_NAME}"
+    DESTINATION_BRANCH = (env.PULL_BASE_REF) ?: main
+    SOURCE_REPO = env.DESTINATION_REPO
```

Comment:
This makes source_repo and destination_repo always the same?

Reply:
ghprb used to propagate the source repo but prow jenkins operator doesn't provide this ! I was trying to make the less changes but a refactoring of the pipeline could be the proper way to handle this so we eliminate where ever this source_repo used

Reply:
done